examples/win/getopt.h

```diff
@@ -0,0 +1,656 @@
+#ifndef __GETOPT_H__
+/**
+ * DISCLAIMER
+ * This file is part of the mingw-w64 runtime package.
+ *
+ * The mingw-w64 runtime package and its code is distributed in the hope that it 
+ * will be useful but WITHOUT ANY WARRANTY.  ALL WARRANTIES, EXPRESSED OR 
+ * IMPLIED ARE HEREBY DISCLAIMED.  This includes but is not limited to 
```

Comment:
why is this added to this repo? @kkzi

Reply:
to support windows with msvc. 
there is  no <getopt.h> in msvc toolchain.

Reply:
In that case it should move to `include/win/getopt.h`

Reply:
Actually not needed since this is only used by examples for now.